.gitignore

```diff
@@ -8,3 +8,8 @@ Rakefile.old
 linked
 /libyjpagent.so
 /.project
+
+lib/jrjackson_jars.rb
+lib/com/fasterxml/jackson/
+lib/jrjackson/jars/jrjackson-*.jar
+pom.xml
```

Comment:
are you sure you want that in the .gitignore?

Reply:
yeah I **think** so. They are only needed during the package phase. AFACT they don't need to be in Github. No?

Reply:
👍